packages/divider-curve/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@littlespoon/divider-curve",
```

Comment:
You will also need to add divider to `@littlespoon/ui`

Reply:
Added